snooty/rstspec-consolidated.toml

```diff
@@ -0,0 +1,1781 @@
+
+[role.azure]
+help = "Link to the Microsoft Azure documentation"
+type = {link = "https://docs.microsoft.com/en-us/azure%s"}
+
+[role.gettingstarted]
+help = """Link to the MongoDB Getting Started guide."""
+type = {link = "https://docs.mongodb.org/getting-started%s", ensure_trailing_slash = true}
+
+[role.master]
+help = """Link to a path in the master branch of the current project."""
+type = "explicit_title"
+
+[role.docsgithub]
+help = """Link to a MongoDB Manual GitHub path."""
+type = {link = "http://github.com/mongodb/docs/blob/master/%s"}
+
+[role.guides]
+help = """Link to a page in the MongoDB Guides site."""
+type = {link = "https://www.mongodb.com/docs/guides%s", ensure_trailing_slash = true}
+
+[role.mms-docs]
+help = """Link to a page in the Cloud Manager documentation."""
+type = {link = "https://www.mongodb.com/docs/cloudmanager%s", ensure_trailing_slash = true}
+
+[role.mms-home]
+help = """Link to a MongoDB Cloud Services page."""
+type = {link = "https://cloud.mongodb.com%s"}
+
+[role.osb]
+help = """Link to a page in the Atlas Open Service Broker documentation."""
+type = {link = "https://www.mongodb.com/docs/atlas-open-service-broker/current%s", ensure_trailing_slash = true}
+
+[role.adl]
+help = """Link to a page in the Atlas Data Lake documentation."""
+type = {link = "https://www.mongodb.com/docs/datalake%s", ensure_trailing_slash = true}
+
+[role.dochub]
+help = """Specify a dochub link."""
+type = {link = "https://dochub.mongodb.org/core/%s"}
+
+[role.mongo-spark]
+help = """Link to a page in the MongoDB Spark Connector GitHub project."""
+type = {link = "https://github.com/mongodb/mongo-spark%s"}
+
+[role.source]
+help = """Link to a file in the MongoDB server GitHub project."""
+type = {link = "https://github.com/mongodb/mongo/blob/master/%s"}
+
+[role.ref]
+help = """Link to a named target."""
+type = {domain = "std", name = "label", format = []}
+
+[role."mongodb:ref"]
+inherit = "ref"
+
+[role."mongodb:card-ref"]
+help = """Create a reference, embedded within cards, with additional styling, that links to a named target."""
+example = """:card-ref:`Android <android-intro>`"""
+inherit = "ref"
+
+[role."mongodb:required"]
+type = "explicit_title"
+
+[role.term]
+help = """Link to a term in the glossary."""
+type = {domain = "std", name = "term", format = []}
+
+[role.issue]
+help = """Link to a JIRA ticket."""
+type = {link = "https://jira.mongodb.org/browse/%s"}
+
+[role.technical-support]
+help = "Link to the MongoDB support site."
+type = {link = "https://www.mongodb.com/docs/manual/support%s", ensure_trailing_slash = true}
+
+[role.contact]
+help = """Link to the MongoDB contact page."""
+type = {link = "https://www.mongodb.com/contact%s"}
+
+[role.mdbacct]
+help = """Link to the MongoDB Account page."""
+type = {link = "https://account.mongodb.com/account%s"}
+
+[role.community-forum]
+help = "Link to the MongoDB Community Forums page."
+type = {link = "https://community.mongodb.com/%s"}
```

Comment:
this should apparently be `https://www.mongodb.com/community/forums/%s` now

Reply:
Done!